multi_agent_research_assistant_ag2/tests/test_agent_setup.py

```
@@ -0,0 +1,31 @@
+    """Verify agent setup does not raise — no LLM calls made."""
+    from research_assistant import build_llm_config
+    from autogen import AssistantAgent, UserProxyAgent, GroupChat, GroupChatManager
+
+    llm_config = build_llm_config()
+    researcher = AssistantAgent(name="researcher", system_message="Test", llm_config=llm_config)
+    analyst = AssistantAgent(name="analyst", system_message="Test", llm_config=llm_config)
+    writer = AssistantAgent(name="writer", system_message="Test", llm_config=llm_config)
+    executor = UserProxyAgent(name="executor", human_input_mode="NEVER", code_execution_config=False)
+
+    gc = GroupChat(agents=[executor, researcher, analyst, writer], messages=[], max_round=12)
+    manager = GroupChatManager(groupchat=gc, llm_config=llm_config)
```

Ruff (0.15.5)

[error] Local variable `manager` is assigned to but never used

Remove assignment to unused variable `manager`

(F841)